wait_for_db.sh

```
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+host="$1"
+shift
+cmd="$@"
+
+POSTGRES_USER="${POSTGRES_USER:-cinema}"
+POSTGRES_DB="${POSTGRES_DB:-cinema}"
+POSTGRES_PASSWORD="${POSTGRES_PASSWORD:-cinema}"
```

Defaulting POSTGRES_PASSWORD here is risky for security and may hide misconfiguration. Prefer to fail fast if required env vars are missing or at least avoid echoing the password in logs. For the task, rely on the .env file passed via docker-compose.